app/models/user.rb

```diff
@@ -37,6 +37,10 @@ def claims
     Claim.fetch_all(vaafi_headers)
   end
 
+  def request_claim_decision(claim_id)
+    Claim.request_decision(vaafi_headers, claim_id)
+  end
```

Comment:
Does it still make sense to do this as part of the user model? I've been wondering about this for the document upload.